### DISCUSSION TITLE
Multiple Tables: Sync horizontal scrolling

### DISCUSSION THREAD

**Discussion by busyxiang** [arco-design/arco-design#886] [Q&A]
Hi, I would like to ask if it is possible to sync horizontal scrolling for multiple tables that use the same columns with different data sets. Tried to use `useRef` on the `Table` component, but it only returns undefined when I console log out the value from `tableRef.current`. Any suggestions or ideas would be much appreciated

**Comment by PengJiyuan** [comment]
```js
function Demo() {
  const ref = useRef(null);
  useEffect(() => {
    console.log(ref.current.getRootDomElement());
  }, []);
  return <Table ref={ref} columns={columns} data={data} />;
}
```

You can use `ref.current.getRootDomElement` to get root element, then find the scroll container.
